rtm/plotting.py

```diff
@@ -112,6 +112,26 @@ def plot_time_slice(S, processed_st, time_slice=None, label_stations=True,
 
     slice = S.sel(time=time_to_plot, method='nearest')
 
+    #convert UTM grid/etc to x/y coordinates with (0,0) as origin
+    if S.UTM and dem is not None:
```

Comment:
Why do we only shift the coordinates if `dem is not None` ? Don't we want the same behavior if no DEM?

Reply:
I was thinking we would only want to do it when a DEM is provided, otherwise it would be a regional case? Perhaps not. Another option is to add an argument for how far to plot/crop from the grid center. It could default to None. I might like that option better.

Reply:
`S.UTM` determines whether a grid is projected (local) or just lat-lon (regional). I think we'd want to treat the projected cases w/o DEM same as those w/ DEM, no?

Reply:
Oh, I see, so now the xy_plot kwarg controls this. Smart.